tools/depcheck/pkg/graph/graph.go

```diff
@@ -99,7 +104,41 @@ func (g *MutableDirectedGraph) PruneOrphans() []*Node {
 		return []*Node{}
 	}
 
-	return append(removed, g.PruneOrphans()...)
+	return append(removed, g.PruneOrphans(ignore)...)
+}
+
+// HasPathFromTo receives a source node A and a destination node B
+// and determines if a path exists between the two nodes.
+// Returns false if A == B
+func (g *MutableDirectedGraph) HasPathFromTo(A, B *Node) bool {
```

Comment:
this looks dead.

Reply:
This is used by the `analyze` command. Will move to that PR